contracts/adapters/berezka/BerezkaTokenAdapterGovernance.sol

```diff
@@ -40,7 +40,7 @@ struct TypedToken {
  */
 contract BerezkaTokenAdapterGovernance is Ownable() {
 
-    AdapterRegistry immutable adapterRegistry = AdapterRegistry(0x06FE76B2f432fdfEcAEf1a7d4f6C3d41B5861672);
+    AdapterRegistry constant adapterRegistry = AdapterRegistry(0x06FE76B2f432fdfEcAEf1a7d4f6C3d41B5861672);
```

Comment:
`constant`'s name should be in UPPER_CASE_WITH_UNDERSCORES
https://solidity.readthedocs.io/en/v0.7.1/style-guide.html#constants

Reply:
also, visibility level is required

I'd recommend to use `internal` in order not to generate getter function named in upper case